### PR TITLE
Created production build and updated README

### PR DESCRIPTION
**Pull request by yume-ind**
This PR adds a static webpage so that normal users can access the GUI more easily.

### CODE REVIEW

File: gui/README.md [hunk 1]
```diff
@@ -8,30 +8,41 @@ npm install
 ### To run the GUI with Cassy
```
Comment (feeblefakie) [inline gui/README.md:8]: Can you update the top title to `Cassy GUI` ?
(sorry I can't comment on non-changed parts)

File: gui/src/api/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export const api = axios.create({
-  baseURL: 'http://127.0.0.1:8090/v1',
+  baseURL: 'http://127.0.0.1:8080/v1',
```
Comment (feeblefakie) [inline gui/src/api/index.js:4]: Is it OK to specify `127.0.0.1` ?

File: gui/README.md [hunk 3]
```diff
@@ -8,30 +8,41 @@ npm install
...
 ```
-npm run serve
+./entrypoint
 ```
-If you have followed the steps correctly, you should have
-1) A running `cassy-server`
-2) A running `entrypoint` on localhost:8090
-3) A running dev server on localhost:8080
 
 In your browser, you can go to [localhost:8080](http://localhost:8080) to view the gui.
```
Comment (feeblefakie) [inline gui/README.md:19]: I think it is `localhost`  when you run this in local so it needs to be updated ?

File: gui/src/api/index.js (hunk 2 above)
Comment (yume-ind) [reply gui/src/api/index.js:4]: You're right, it should probably be `localhost`.

File: gui/README.md (hunk 3 above)
Comment (yume-ind) [reply gui/README.md:19]: Sorry, could you explain more what you mean?

Comment (feeblefakie) [reply gui/README.md:19]: I meant, does it depend on where you run Cassy ?

Comment (yume-ind) [reply gui/README.md:19]: The GUI will be served on `localhost` no matter where you run `cassy-server`. I haven't tried running `entrypoint` with Cassy running at a different IP address, but it shouldn't affect the GUI as far as I understand.

File: gui/src/api/index.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export const api = axios.create({
-  baseURL: 'http://127.0.0.1:8090/v1',
+  baseURL: 'http://localhost:8080/v1',
```
Comment (feeblefakie) [inline gui/src/api/index.js:4]: So, it should be reverted ?

Comment (yume-ind) [reply gui/src/api/index.js:4]: I guess so. Hopefully it doesn't cause issues.

**Comment by yume-ind** [comment]
@feeblefakie I think this PR should be okay to merge, what do you think? I reverted the change to `localhost` for the time being.

Once this PR is merged, I can create another one for the Cassy backup script.

**Review by feeblefakie — APPROVED** [review]
LGTM ! Sorry for late.

**Pull request merged**